public/index.html

```diff
@@ -0,0 +1,370 @@
+                </thead>
+                <tbody>
+                    <tr>
+                        <td class="table-price">$25.99/MONTH</td>
+                    </tr>
+                    <tr>
+                        <td>15 Projects</td>
+                    </tr>
+                    <tr>
+                        <td>50 Users</td>
+                    </tr>
+                    <tr>
+                        <td>100 Gb user</td>
+                    </tr>
+                    <tr>
+                        <td>25 Calendar </td>
+                    </tr>
+                    <tr class="table-foot">
+                        <td>
+                            <button type="button" class="button-signup">SIGN UP</button>
+                        </td>
+                    </tr>
+                    <tbody>
+            </table>
+            <table class="flex-item-3">
+                <thead>
+                    <tr class="table-head">
+                        <th>PREMIUM</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td class="table-price">$49.99/MONTH</td>
+                    </tr>
+                    <tr>
+                        <td>UNLIMITED Projects</td>
+                    </tr>
+                    <tr>
+                        <td>UNLIMITED Users</td>
+                    </tr>
+                    <tr>
+                        <td>1000 Gb user</td>
+                    </tr>
+                    <tr>
+                        <td>UNLIMITED Calendar </td>
+                    </tr>
+                    <tr class="table-foot">
+                        <td>
+                            <button type="button" class="button-signup">SIGN UP</button>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
+    </section>
+    <section class="section-white">
+        <object class="clip clip-orange" type="image/svg+xml" data="./imgs/cliporange.svg"></object>
+        <a id="portfolio"></a>
+        <h1>PORTFOLIO</h1>
+        <div class="carousel_portfolio">
+            <img class="carousel-lrg" src="./imgs/portfolio.jpg" alt="hand holding phone in city">
+        </div>
+        <nav>
+            <ul class="portfolio-nav">
+                <li><a href="#portfolio" class="selected selected-orange">All works</a></li>
+                <li><a href="#portfolio">Webdesign</a></li>
+                <li><a href="#portfolio">UI design</a></li>
+                <li><a href="#portfolio">Icons design</a></li>
+                <li><a href="#portfolio">Brand &#38; Logotypes</a></li>
+            </ul>
+        </nav>
+        <ul class="flex-container">
+            <li class="flex-item-2">
+                <img src="./imgs/youtube.jpg" alt="computer screen">
+                <p><strong>Youtube iphone app</strong>
+                    <br>UI design, brand &#38; logotypes</p>
```

Comment:
Also, I think this is a textbook example of the use case for `figure` and `figcaption`.